src/containers/email-confirm-modal/EmailConfirmPopupSuccess.jsx

```
@@ -0,0 +1,23 @@
+import Box from '@mui/material/Box'
+import AppButton from '~/components/app-button/AppButton'
+import { styles } from '~/containers/email-confirm-modal/EmailConfirmModal.styles'
+import ImgTitleDescription from '~/components/img-title-description/ImgTitleDescription'
+import imgSuccess from '~/assets/img/email-confirmation-modals/success-icon.svg'
+import { useTranslation } from 'react-i18next'
+
+const EmailConfirmPopupSuccess = ({openLoginDialog}) => {
```

ESLint

[error] Replace `openLoginDialog` with `·openLoginDialog·`

(prettier/prettier)